ui/js/directives/treeherder/clonejobs.js

```diff
@@ -958,9 +955,9 @@ treeherder.directive('thCloneJobs', [
         };
         var generateJobElements = function(resultsetAggregateId, resultset) {
```

Comment:
Lots of function in this file were poorly named.  Probably by me at some point...  :)  This function might be more aptly named as ``generateJobTable``.  Or can you think of a better name?